configs/gnmic/gnmic-config.yml

```diff
@@ -3,127 +3,116 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 username: admin
-password: NokiaSrl1!
-port: 57400
 timeout: 10s
-skip-verify: true
-encoding: json_ietf
+
+common_srl_subscriptions: &common_srl_subs
+  port: 57400
+  skip-verify: true
+  insecure: false
+  password: NokiaSrl1!
```

Comment:
I think these could stay at the top of the config, as they are connection details

Reply:
I prefer it like this, with that its easier to add sros if needed. For example:

`username: admin
password: *******
insecure: true
log: true

common_sros_subscriptions: &common_sros_subs
  insecure: true
  password: ******
  subscriptions:
      - sros-if-stats
      - sros-bgp-stats
      - sros-service-stats

common_srl_subscriptions: &common_srl_subs
  port: 57400
  skip-verify: true
  insecure: false
  password: *******
  subscriptions:
      - srl-if-stats 
      - srl-bgp
      - srl-system-performance
      - srl-routes
      - srl-bridge
      - srl-apps
      - srl-net-instance`